team-01/src/main/java/ubc/cosc322/MonteCarloPlayer.java

```diff
@@ -118,20 +119,27 @@ public TreeNode(AmazonsLocalBoard state, AmazonsAction action, TreeNode parent)
 			children = new ArrayList<TreeNode>();
 		}
 
-		// TODO
-
 		/**
 		 * Returns the first child expanded.
 		 */
 		public TreeNode expand() {
 			// Get list of possible actions
+			ArrayList<AmazonsAction> actions = actionFactory.getActions(state);
 
 			// Make new state node for each action
+			for (int i = 0; i < actions.size(); i++) {
+				AmazonsAction childAction = actions.get(i);
+
+				AmazonsLocalBoard childState = state.copy();
+				childState.localPlayer = state.localPlayer == 1 ? 2 : 1;
```

Comment:
Perhaps it's simpler to say 
```suggestion
				childState.localPlayer = 3 - state.localPlayer;
```
?

Reply:
In this case, I found it more readable to write it long-form, since 3 doesn't really have an obvious meaning here.